### ISSUE TITLE
 Unable to create pre-release with an empty token

### ISSUE THREAD

**Issue by vincent-herlemont** [python-semver/python-semver#344]
# Situation

Unable to create pre-release with an empty token.

# To Reproduce

Run this script:
```python
ver = semver.VersionInfo.parse('1.0.0')
ver = ver.bump_prerelease('');
print(ver)
# 1.0.0-rc.1
```

# Expected Behavior

`.bump_prerelease('')` should generate an version like `1.0.0-1`.

# Environment
- OS: Linux ubuntu 5.11.0-40-generic
- Python version: Docker image python:3.8
- Version of semver library: semver==2.13.0

# Additional context

Pre-release documentation https://semver.org/#spec-item-9

**Comment by tomschr** [comment]
Thank you @vincent-herlemont, much appreciated. Sorry for the late response.

I suppose, this is also true for the method `.bump_build()`.

**Comment by tomschr** [comment]
@vincent-herlemont One question. What would you expect if you have already a prerelease and pass an empty string? Like this:

```
>>> ver = semver.VersionInfo.parse('1.0.0-rc.1')
>>> ver.bump_prerelease('')
# ???
```

I could think of two solutions:

1. Basically ignore the token argument in `.bump_prerelease()`. In our case above, you would get `1.0.0-rc.2`.
2. Overwrite with the token argument in `.bump_prerelease()`. In our case above, you would get `1.0.0-0`.

If you already have a prerelease, you would like to continue when using `.bump_prerelease()`, right? Solution 2 would be more flexible.

In other words: If prerelease isn't set, the token is taken into account, otherwise it's ignored (regardless of the value).
